Makefile

```diff
@@ -47,6 +47,12 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Default k4e-Operator  namespace
+K4E_OPERATOR_NAMESPACE ?= "k4e-operator-system"
```

Comment:
There are a lot of places in the makefile that use the explicit name of the namespace 
maybe a good refactoring is to replace all of them... 
Also, there is assumption in our code that the default namespace called "k4e-operator-system" (for example main.go and https://github.com/jakub-dzon/k4e-operator/blob/main/config/default/kustomization.yaml#L2)
There is a way to reuse that parameter?

Reply:
For the main.go I do not think so. This way only uses one variable instead of multiple. 
For the makefile, I cannot find where you're referring to. The idea to have in a variable is to be able to do `make run  K4E_OPERATOR_NAMESPACE=newNS`
For the kustamization, that it's a variable AFAIK, no?

Reply:
You're right I looked into the README file, my mistake

Reply:
:+1: